src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -11890,7 +11890,8 @@ void Unit::UpdateSpeed(UnitMoveType mtype)
             else             // Use not mount (shapeshift for example) auras (should stack)
                 main_speed_mod  = GetTotalAuraModifier(SPELL_AURA_MOD_INCREASE_FLIGHT_SPEED) + GetTotalAuraModifier(SPELL_AURA_MOD_INCREASE_VEHICLE_FLIGHT_SPEED);
 
-            non_stack_bonus += GetMaxPositiveAuraModifier(SPELL_AURA_MOD_FLIGHT_SPEED_NOT_STACK) / 100.0f;
+            // SPELL_AURA_MOD_MOUNTED_SPEED_NOT_STACK should increase flying mount speed as well.
+            non_stack_bonus += std::max(GetMaxPositiveAuraModifier(SPELL_AURA_MOD_FLIGHT_SPEED_NOT_STACK), GetMaxPositiveAuraModifier(SPELL_AURA_MOD_MOUNTED_SPEED_NOT_STACK)) / 100.0f;
```

Comment:
Not sure about this.

From 4.3.4 Spell.dbc:
```
ID - 26022 Pursuit of Justice (Rank 1)
=================================================
Effect 0: Id 6 (SPELL_EFFECT_APPLY_AURA)
BasePoints = 8
Targets (1, 0) (TARGET_UNIT_CASTER, NO_TARGET)
Aura Id 31 (SPELL_AURA_MOD_INCREASE_SPEED), value = 8, misc = 0 (0), miscB = 0, periodic = 0

Effect 1: Id 6 (SPELL_EFFECT_APPLY_AURA)
BasePoints = 8
Targets (1, 0) (TARGET_UNIT_CASTER, NO_TARGET)
Aura Id 172 (SPELL_AURA_MOD_MOUNTED_SPEED_NOT_STACK), value = 8, misc = 0 (0), miscB = 0, periodic = 0

Effect 2: Id 6 (SPELL_EFFECT_APPLY_AURA)
BasePoints = 8
Targets (1, 0) (TARGET_UNIT_CASTER, NO_TARGET)
Aura Id 211 (SPELL_AURA_MOD_FLIGHT_SPEED_NOT_STACK), value = 8, misc = 0 (0), miscB = 0, periodic = 0
```

Clearly this one has both `SPELL_AURA_MOD_MOUNTED_SPEED_NOT_STACK` and `SPELL_AURA_MOD_FLIGHT_SPEED_NOT_STACK`, so the original code seems legit to me